.travis.yml

```diff
@@ -52,9 +52,9 @@ before_install:
   - travis_retry pip uninstall click -y
 
 install:
-  - pip install -r requirements-dev.txt
-  - travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt
-  - travis_retry pip install -e .[$EXTRAS]
+  - pip install --ignore-installed -r requirements-dev.txt
```

Comment:
Could you use `-I` to keep consistency on shorter variants?

Reply:
@Sinclert Great to see you contributing again :smile: Can you please also rebase and squash the various commits together?

Reply:
Happy to help 😃 

I changed `--ignore-installed` to `-I`, and squashed commits into 3 (_cli_,  _installation_, and _travis_). I could have squashed all of them together but it seems that they affect different scopes of the repository.